### ISSUE TITLE
Manual states incorrect box size when describing `\pgfsetfadingforcurrentpath`

### ISSUE THREAD

**Issue by ghost** [pgf-tikz/pgf#868]
I think that the pgfmanual v3.1.5 is not entirely correct when describing `\pgfsetfadingforcurrentpath` and `\pgfsetfadingforcurrentpathstroked` on p. 1202. It states that 

> Otherwise it is assumed that the fading has a size of 100bp times 100bp.

However, if one looks these functions up one finds in lines

https://github.com/pgf-tikz/pgf/blob/65bcaafffbd0b21974b085504b4853684d5ed8bc/tex/generic/pgf/basiclayer/pgfcoretransparency.code.tex#L194-L195

and

https://github.com/pgf-tikz/pgf/blob/65bcaafffbd0b21974b085504b4853684d5ed8bc/tex/generic/pgf/basiclayer/pgfcoretransparency.code.tex#L238-L239

So the actual box size is 50pt times 50pt (or 50bp times 50bp). This might be consistent with the statements on p. 1097 (if one replaces `bp` by `pt`).

Note that the claims here have been "empirically confirmed" in https://tex.stackexchange.com/a/546801/194703.

**Comment by hmenke** [comment]
To be honest, I don't really understand the low-level code here.  I guess I have to research more. On a side note, 50 pt ≠ 50 bp.

**Comment by ghost** [comment]
@hmenke  Yes, I know that 50bp is not precisely 50pt, just wanted to avoid comments saying I am too picky since they are very similar. The low-level code computes the width and height of the path picture, and finds a transformation that rescales a rectangle of width 50pt and height 50pt to the size of the path picture, and translates you to the center of the path picture (or however you want to call it).  Imagine that before `\pgf@xc=.02\pgf@xc` the length `\pgf@xc`, which is then defined as the difference `\pgf@pathmaxx-\pgf@pathminx`, equals 50pt. After `\pgf@xc=.02\pgf@xc` we will have `\pgf@xc=1pt`, and the 11 entry of the transformation `\pgftransformcm{\pgf@sys@tonumber{\pgf@xc}}{0}{0}{\pgf@sys@tonumber{\pgf@yc}}{\pgfpointorigin}%` will be 1, so trivial. If the width is `f*50pt`, then 11 transformation matrix entry will be `f`. Likewise for the 22 entry, which rescales the y direction.
